src/System.Windows.Forms.Design/src/System/ComponentModel/Design/DesignerActionUIService.cs

```diff
@@ -33,34 +31,34 @@ public void Dispose()
     {
         if (_serviceProvider is not null)
         {
-            IDesignerHost host = (IDesignerHost)_serviceProvider.GetService(typeof(IDesignerHost));
+            IDesignerHost? host = (IDesignerHost?)_serviceProvider.GetService(typeof(IDesignerHost));
             host?.RemoveService<DesignerActionUIService>();
         }
     }
 
     /// <summary>
     ///  This event is thrown whenever a request is made to show/hide the UI.
     /// </summary>
-    public event DesignerActionUIStateChangeEventHandler DesignerActionUIStateChange
+    public event DesignerActionUIStateChangeEventHandler? DesignerActionUIStateChange
     {
         add => _designerActionUIStateChangedEventHandler += value;
         remove => _designerActionUIStateChangedEventHandler -= value;
     }
 
-    public void HideUI(IComponent component)
+    public void HideUI(IComponent? component)
     {
         OnDesignerActionUIStateChange(new DesignerActionUIStateChangeEventArgs(component, DesignerActionUIStateChangeType.Hide));
     }
```

Comment:
Let's update this to use body expression
```suggestion
    public void HideUI(IComponent? component) => OnDesignerActionUIStateChange(new DesignerActionUIStateChangeEventArgs(component, DesignerActionUIStateChangeType.Hide));
```